### ISSUE TITLE
Jupyter notebook run inline

### ISSUE THREAD

**Issue by divyaramamoorthy** [maalcantar/bedl_jupyter_notebook_resources#4]
In FAQs - could mention how to use the below code block to display all outputs from a cell
```
%matplotlib inline
from IPython.core.interactiveshell import InteractiveShell
InteractiveShell.ast_node_interactivity = "all"
```

**Comment by pablocarderam** [comment]
Woah didn't know about this :O

**Comment by jackievaleri** [comment]
Addressed in d079cd5b2bb5867f552002230376a663bd4d7617. Cool tip!
